selfservice/strategy/oidc/provider_discord.go

```diff
@@ -0,0 +1,91 @@
+package oidc
+
+import (
+	"context"
+	"fmt"
+	"github.com/bwmarrin/discordgo"
+	"github.com/ory/herodot"
+	"github.com/ory/x/stringslice"
+	"github.com/ory/x/stringsx"
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+	"net/url"
+)
```

Comment:
Looks like you didn't run `make format`?

Reply:
Yeah... `make format` seems to use lintx...when I try to install lintx it wants to install setupdocx, which has a dependency on setuplibs which has (how it looks to me) again a circular dependency on setupdocx... I think i have to do this tomorrow.

Reply:
Normally my IDE formats everything correct but not this time haha

Reply:
you have to
```
$ go install github.com/sqs/goreturns
```

goreturns just orders imports differently than the default by grouping them

Reply:
Done. Somehow `github.com/ory/x/tools/listx` was not installed and I thought lintx was [this](https://pypi.org/project/listx/). But it seems it is supposed to be installed automatically?

Reply:
There are some hacks in the Makefile but it does not work for me reliably either....